cwms_radar_api/src/main/java/cwms/radar/api/graph/basinconnectivity/nodes/BasinConnectivityNode.java

```diff
@@ -0,0 +1,87 @@
+package cwms.radar.api.graph.basinconnectivity.nodes;
+
+import cwms.radar.api.graph.Node;
+import cwms.radar.api.graph.basinconnectivity.BasinConnectivityStream;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.*;
+
+public abstract class BasinConnectivityNode implements Node
+{
+    private final String streamId;
+    private final Double station;
+    private final String bank;
+
+    protected BasinConnectivityNode(String streamId, Double station, String bank)
```

Comment:
NOT Required: would it make sense to just have two constructors and have this one error on a null bank?

probably doesn't matter much since it's only used by the builder but food for thought.